cmd/tempo/app/app.go

```diff
@@ -54,6 +54,7 @@ const (
 )
 
 var (
+	// TODO: remove this metric?
```

Comment:
yeah, i'd go ahead and remove this. it looks like it only existed for serverless for some reason